app/services/reporting/total_user_count_report.rb

```diff
@@ -44,18 +57,40 @@ def verified_user_count
       end
     end
 
+    def new_verified_user_count
+      Reports::BaseReport.transaction_with_timeout do
+        Profile.where(active: true).where(activated_at: current_month).count
+      end
+    end
+
     def annual_total_user_count
       Reports::BaseReport.transaction_with_timeout do
-        User.where(created_at: annual_start_date..end_date).count
+        User.where(created_at: annual_start_date..annual_end_date).count
+      end
+    end
+
+    def annual_verified_user_count
+      Reports::BaseReport.transaction_with_timeout do
+        Profile.where(active: true).
+          where(activated_at: annual_start_date..annual_end_date).
```

Comment:
Same question as before, though maybe less of an edge case over the course of a year.